### ISSUE TITLE
Buff Legion in the new map

### ISSUE THREAD

**Issue by Bodynasr** [BadDeathclaw/TG-Claw#1014]
Buff the Legion, **they barely won a single round** since the new map kicked in. Everyone notices it and people stopped playing the legion due to it.
![legion](https://user-images.githubusercontent.com/51898651/59590748-09593180-90ed-11e9-96ed-2dd1a42a245a.PNG)

Armory: 
-Add more shit tier weapons such as Ithacas and scoped hunting rifles. WHY WERE BOLAS REMOVED!?

-MORE AMMUNITION, for a faction where almost all use revolvers, the revolver ammo runs dry by min 30 mark. Return the low power 7.62 ammo back

-Add a limited amount of Molotovs? NCR got flashbangs as well

-Bring back C4, we are tired of having to go into a frontal assault against the meta of Bonfires
TLDR: **RETURN** IT LIKE IT USED TO BE. Every NCR recruit takes a service rifle, extra clips and a side arm with its clips as well.

General problems of the legion:

-The bonfire meta, Both NCR and legion puts bonfire everywhere and for a close range/melee faction, it simply kills the legion. I suggest deleting it, or make the fire wear out much quicker. It simply breaks immersion, take a look at new vegas, do you see any of the factions fortifications have bonfires in it? 

-Advanced Signaling devices, we used to have three of them but now we just have one? Why? Some dude runs off with one and now all slave collars have become useless.

It was a long rant but after 12 rounds of having our asses whooped hard, something needs to be done.

**Comment by AdamElTablawy** [comment]
after 12 rounds of aving your asses whooped hard, something does need to be done

you need to get good, is what you need to do

take a pickaxe and fucking mine into the base you dumbass, you keep attacking from the front and you learned the hard way that doesn't work on sunnydale either

**Comment by BigManClancy** [comment]
- Bolas were readded.
- More .44 speedloaders were added
- Go craft the molotovs yourself you lazyass
- C4's are not coming back, just take one of the pickaxes i have provided to the Legion base and start laboring into the NCR base from the side and stop trying frontal assaults.
- Bonfire meta is kill soon, do not worry
- I added a 2nd Advanced Signaler. Make sure your Centurion has one and then somebody else can have the other one.

This might aswell be closed now as the issues have been adressed.
